src/actions/conversation.js

```diff
@@ -31,14 +34,27 @@ export const getConversations = ({
   assigneeType,
   conversationStatus,
   inboxSelected,
+  pageNumber,
 }) => async dispatch => {
   dispatch({ type: GET_CONVERSATION });
   try {
+    let assignee;
+    switch (assigneeType) {
+      case 0:
+        assignee = 'me';
```

Comment:
`me`, `unassigned ` and `all` are used in more than one place, Making it constant will be better.  The same is implemented here `unassigned`.